repository/Patient.js

```diff
@@ -14,14 +14,59 @@ class Patient {
       transaction
     );
 
-    return await model.create(
+    return await patientModel.create(
       {
         ...field,
         userId: user.id,
       },
       transaction
     );
   }
+
+  /**
+   * @description This controller is used to get a patient by their Id
+  */
+  static async getPatient(field = {}) {
+    const { id } = field;
+    return await patientModel.findOne(
+      {
+        where: {
+          id
+        },
+        include: [
+          {
+            model: userModel,
+            as: 'login_details',
+            attributes: ['email']
+          }
+        ]
+      }
+    )
+
+  }
+
+  static async searchForPatient(name = '') {
+
+    return await patientModel.findAll({
```

Comment:
Remove line 49. There is no need for the new line.
Put back the transaction here. It is only in the controller that you should remove the transaction.